### PR TITLE
Feature/up styling aside menu

### PR DESCRIPTION
**Pull request by Dragonesis**

### CODE REVIEW

File: src/components/MenuPanel/WalletManager/Details/Addresses/Item/Item.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <div className='menu-panel-wallet-manager-details-addresses-item' key={address}>
+      <JRadio
+        onChange={console.log()}
+        label={addressName ? (
+          <MenuPanelWalletManagerDetailsAddressWithName
+            address={getShortenedAddress(address, SYMBOLS_FROM_START, SYMBOLS_FROM_END)}
+            addressName={addressName}
+          />
+        ) : (
+          getShortenedAddress(address, SYMBOLS_FROM_START, SYMBOLS_FROM_END)
```
Comment (IViol) [inline src/components/MenuPanel/WalletManager/Details/Addresses/Item/Item.js]: write this value in re-usable var

File: src/components/MenuPanel/WalletManager/Details/Addresses/Item/Item.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        )}
+        name='address'
```
Comment (IViol) [inline src/components/MenuPanel/WalletManager/Details/Addresses/Item/Item.js]: name={address}

File: src/components/MenuPanel/WalletManager/Details/Addresses/addresses.scss [hunk 3]
```diff
@@ -1,4 +1,12 @@
+@import './Item/item';
+
 .menu-panel-wallet-manager-details-addresses {
   all: initial;
-  height: 135px;
+  height: 155px;
+  padding-left: 3px;
```
Comment (IViol) [inline src/components/MenuPanel/WalletManager/Details/Addresses/addresses.scss]: padding: 20px 0 20px 3px

File: src/components/MenuPanel/WalletManager/Details/details.scss [hunk 4]
```diff
@@ -13,11 +13,12 @@
   overflow: hidden;
   opacity: 0;
   will-change: opacity, max-height;
+  padding-left: 50px;
+  padding-right: 50px;
 
   > .action {
-    @include flex-row-center;
-
-    padding: 20px 0;
```
Comment (IViol) [inline src/components/MenuPanel/WalletManager/Details/details.scss]: why?

File: src/components/MenuPanel/WalletManager/Main/main.scss [hunk 5]
```diff
@@ -26,11 +30,21 @@
   }
 
   > .chevron {
-    margin: 0 -4px 0 auto;
```
Comment (IViol) [inline src/components/MenuPanel/WalletManager/Main/main.scss]: margin-left should be `auto`

Comment (Dragonesis) [reply src/components/MenuPanel/WalletManager/Main/main.scss]: why?

**Pull request merged**